### PR TITLE
Demo page for Litle (Vantiv) Plugin

### PR DESCRIPTION
**Pull request by javier-gomez-github-zz**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -0,0 +1,24 @@
+GEM
```
Comment (pierre) [inline Gemfile.lock]: Nit: I don't think you need to check this file in (see https://github.com/killbill/killbill-stripe-demo/blob/master/.gitignore#L2).

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+  subscription.price_overrides = []
+
+  # For the demo to be interesting, override the trial price to be non-zero so we trigger a charge in Litle
+  override_trial = KillBillClient::Model::PhasePriceOverrideAttributes.new
+  override_trial.phase_type = 'TRIAL'
+  override_trial.fixed_price = 10.0
+  subscription.price_overrides << override_trial
+
+  begin
+    # sometime returns an error: "Error locking accountRecordId='***'"
```
Comment (pierre) [inline app.rb]: This is odd / should never happen. Do you have more details?

File: app.rb [hunk 3]
```diff
@@ -0,0 +1,300 @@
...
+                <tr><td>Paypage Registration ID</td><td><input type="text" id="paypageRegistrationId" name="paypageRegistrationId" readOnly="true"/> <--Hidden</td></tr>
+                <tr><td>Bin</td><td><input type="text" id="bin" name="bin" readOnly="true"/> <--Hidden</td></tr>
+                <tr><td></td><td align="right"><input type="submit" id="submitId">Check out</button></td></tr>
+            </table>
+        </form>
+        <br/>
+        <h3>Test Input Fields</h3>
+        <table class="testFieldTable">
+            <tr>
+                <td>Paypage ID</td><td><input type="text" id="request$paypageId" name="request$paypageId" value="a2y4o6m8k0" disabled/></td>
```
Comment (pierre) [inline app.rb]: This should be configurable as it depends on the merchant - maybe introduce an environment variable like `KB_URL` (and update the README)?

**Review by pierre — COMMENTED** [review]
Awesome! Just a couple of questions and remarks.

File: app.rb (hunk 2 above)
Comment (javier-gomez-github-zz) [reply app.rb]: Yes, this is not necessary. I based my code in the dwolla-demo and forgot to remove this line.

File: app.rb (hunk 3 above)
Comment (javier-gomez-github-zz) [reply app.rb]: Yes! I wanted you to confirm this before proceeding. I'll add the environment variables and update the README. Thanks @pierre

**Comment by javier-gomez-github-zz** [comment]
@pierre 
Done with fixes! 💪

File: app.rb [hunk 4]
```diff
@@ -52,20 +56,18 @@ def create_subscription(account, user, reason, comment, options)
   override_trial.phase_type = 'TRIAL'
   override_trial.fixed_price = 10.0
   subscription.price_overrides << override_trial
-
-  begin
-    # sometime returns an error: "Error locking accountRecordId='***'"
-    subscription.create(user, reason, comment, nil, true, options)
```
Comment (pierre) [inline app.rb]: You still need to create the subscription!

File: app.rb [hunk 5]
```diff
@@ -147,12 +149,13 @@ def create_subscription(account, user, reason, comment, options)
         <h3>Test Input Fields</h3>
         <table class="testFieldTable">
             <tr>
-                <td>Paypage ID</td><td><input type="text" id="request$paypageId" name="request$paypageId" value="a2y4o6m8k0" disabled/></td>
-                <td>Merchant Txn ID</td><td><input type="text" id="request$merchantTxnId" name="request$merchantTxnId" value="987012"/></td>
+            value=<%= "'#{@customerId}'" %>
```
Comment (pierre) [inline app.rb]: Extra debug statement?

**Review by pierre — CHANGES_REQUESTED** [review]

**Comment by javier-gomez-github-zz** [comment]
@pierre my bad, the christmas spirit 🎅  made me add and delete extra lines 😰

**Comment by pierre** [comment]
Thanks!

**Pull request merged**